docs/index.md

```diff
@@ -4,14 +4,14 @@ Go to the livestream: [stream.hoobs.live](https://stream.hoobs.live/).
 
 
 ## Youtube Live Stream
-* [Chat Commands](../youtube#commands)
-* [Blex Bot](../youtube#blexbot)
+* [Chat Commands](../docs/youtube/index.md)
+* [Blex Bot](../docs/youtube/index.md)
 
 
 ## Minecraft Server
 * [Hoobasaurus Family Lore](lore)
-* [Server Anti-Lag Rules](rules/lag)
-* [How to Join the Server](help/connecting)
+* [Server Anti-Lag Rules](rules/lag.md)
```

Comment:
we dont need the .md

Reply:
We need it otherwise it will show 404 on GitHub.